hmda/src/test/scala/hmda/validation/rules/lar/validity/V665_1Spec.scala

```diff
@@ -0,0 +1,16 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.filing.lar.LarGenerators._
+import hmda.model.filing.lar.LoanApplicationRegister
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.lar.LarEditCheckSpec
+
+class V665_1Spec extends LarEditCheckSpec {
+  override def check: EditCheck[LoanApplicationRegister] = V665_1
+
+  property("Co-applicant Credit score must be valid") {
+    forAll(larGen) { lar =>
+      lar.mustPass
```

Comment:
Does this test cover all cases? (i.e. when should it fail?)

Reply:
Our parser already makes sure this edit will never fail.  The edit checks if the credit score is numeric, and the field type is an Integer.  I can't set the field to be any value that will cause this edit to fail.